config/sources/families/rk35xx.conf

```diff
@@ -24,6 +24,9 @@ case $BRANCH in
 
 		SKIP_BOOTSPLASH="yes"
 		LINUXFAMILY=rk35xx
+		if [[ $BOARD == rock-3a ]]; then
+			BOOT_FDT_FILE="rockchip/rk3568-rock-3a.dtb"
+		fi
```

Comment:
What do you think about doing it the other way round?
What I mean is changing the `BOOT_FDT_FILE` in the board config and moving this conditional (with old dtb name) to legacy segment?
That would keep cruft in the legacy - they will both go away at some point ;-)